### DISCUSSION TITLE
Abbreviated breach-analytics response

### DISCUSSION THREAD

**Discussion by bertold** [XposedOrNot/XposedOrNot-API#8] [Ideas]
The response payload from the breach-analytics endpoint can be large (>10k). Would it be possible to provide a shortened version that would not include all the stats, just the name/date of each breach the email is found in? This would make the server side do less work, too. Thank you.

**Comment by DevaOnBreaches** [comment]
Hello @bertold,

Thank you for your feedback and suggestion regarding the response payload size from the breach-analytics endpoint. Iappreciate your input and are committed to improving our API based on user needs.

I understand your concern about the response payload size, and we can certainly work on providing a shortened version that includes only the essential information, such as the name and date of each breach where the email is found. This optimization will not only reduce the payload size but also ease the load on our server.

Additionally, as you mentioned, there is an alternate API route that already contains the year component. I can explore enhancing this route to include the month component as well, which can offer more detailed information for your needs.

```shell
https://api.xposedornot.com/v1/analytics/[email]
```

We are actively working on these improvements, and your feedback is valuable in guiding our development efforts. Please stay tuned for updates on these enhancements.

If you have any further suggestions or specific requirements, please don't hesitate to let us know. We're here to ensure that our API meets your needs.

Thank you for using our services, and I look forward to delivering an improved experience for you.

**Comment by DevaOnBreaches** [comment]
Moving this to closure with b4b6b12
